### PR TITLE
LPS-124257 Create test to check if the date mask is applied according to the locale

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js [hunk 1]
```diff
@@ -78,19 +59,41 @@ const getDelimiter = (dateFormat) => {
 	return dateDelimiter;
 };
 
-const getDateFormat = () => {
-	const dateFormat = Liferay.AUI.getDateFormat();
-	const dateDelimiter = getDelimiter(dateFormat);
+const getLocaleDateFormat = (locale, format = 'L') => {
+	moment.locale(locale);
```
Comment (alinedoleron) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js:63]: @natocesarrego, I needed to put this line here to fix a bug. But in this case the chaining is not possible.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js [hunk 2]
```diff
@@ -13,50 +13,31 @@
...
-			inputMaskArray.push(/\d/);
-			inputMaskArray.push(/\d/);
-		}
-	});
-
-	return inputMaskArray;
-};
-
-const getDateMask = (dateFormat, dateDelimiter) => {
+const getDateMask = (dateDelimiter, dateFormat) => {
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js:27]: Nice SF, @alinedoleron! 😄

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/DatePicker/DatePicker.es.js [hunk 3]
```diff
@@ -24,6 +24,7 @@ import DatePicker from '../../../src/main/resources/META-INF/resources/DatePicke
 const spritemap = 'icons.svg';
 
 const defaultDatePickerConfig = {
+	locale: 'en_US',
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/DatePicker/DatePicker.es.js:27]: Good improvement! 😄 👏

**Review by natocesarrego — APPROVED** [review]
Gooood job, Alininha! 🎉

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**